src/components/message/message.scss

```diff
@@ -0,0 +1,57 @@
+@use "../../design-tokens/colors.yml";
+@use "../../mixins/focus";
+@use "../../design-tokens/font.yml";
+@use "../../design-tokens/sizes.yml";
+@use "../../mixins/ms";
+@use "../../mixins/unit";
+
+/**
+ * 1. Display message and close button in a centered row.
+ * 2. Push message and close button as far apart as possible.
+ * 3. Condense line height in case it breaks to multiple lines.
+ *
+ * @TODO
+ * 1. Style `:focus` state
```

Comment:
Can this be one line? I spent a few moments trying to figure out how this `@todo` related to comment 1, then I realized [you weren't using sassdoc conventions](http://sassdoc.com/annotations/#todo).

Reply:
Also, I'm not sure what focus state the message would have? The close button already has one. 🤔